packages/components/label/stories/label.stories.ts

```diff
@@ -11,6 +11,9 @@ export default {
     required: {
       description: 'Set label in required mode',
     },
+    readonly: {
+      description: 'Set label in readonly mode',
+    },
```

Comment:
Maybe we can add the default value and type

Reply:
I have updated this part